### PR TITLE
Advance receipt adjustment sanity issues fixed

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ export class ViewTransactionsComponent implements OnInit, OnChanges, OnDestroy {
         let downloadVoucherRequestObject = {
             voucherNumber: [invoice.voucherNumber],
             voucherType: invoice.voucherType,
-            accountUniqueName: (invoice.account) ? invoice.account.uniqueName : ''
+            accountUniqueName: invoice.account.uniqueName
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:149]: Revert this change

Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:149]: why we removed nulity check?
